engine/execution/computation/computer/computer_test.go

```diff
@@ -1253,18 +1253,18 @@ func Test_ExecutingSystemCollection(t *testing.T) {
 	expectedNumberOfEvents := 4
 	expectedMinEventSize := 1000
 
-	// bootstrapping does not cache programs
-	expectedCachedPrograms := 0
+	// bootstrapping already caches some programs
```

Comment:
Due to the changes that the weights are not on the service account, bootstrapping already fills the cache with some programs.

Reply:
That last message was from after you pushed your changes but before I pushed my latest ones. I didn't pull down your changes so I think I overwrote them. Do you know how to fix that?

Reply:
Pushed the fix again.